src/main/java/com/softserve/controller/RoomController.java

```diff
@@ -127,16 +127,19 @@ public ResponseEntity<List<RoomDTO>> getAllOrdered() {
     @PostMapping("/after/{id}")
     @ApiOperation(value = "Create room after id")
     public ResponseEntity<RoomDTO> saveRoomAfterId(@PathVariable("id") Long roomAfterId, @RequestBody RoomDTO roomDTO) {
+        if (roomAfterId == null) {
```

Comment:
path variable are required by default
if it is missing, an exception should be thrown, how can this parameter be null considering that?